src/Collections-Unordered/Bag.class.st

```diff
@@ -132,11 +132,26 @@ Bag >> associationsDo: aBlock [
 	contents associationsDo: aBlock
 ]
 
+{ #category : 'accessing' }
+Bag >> counts [
+	"Returns the amount of occurrences of each element."
+	"#(11 22 22 33 11 11 11) asBag counts >>>  #(4 2 1)"
+	
+	^ contents values
+]
+
 { #category : 'accessing' }
 Bag >> cumulativeCounts [
 	"Answer with a collection of cumulative percents covered by elements so far."
 
-	"#(1 2 2 3 1 1 1) asBag cumulativeCounts >>> {57.1->1 . 85.7->2 . 100.0->3}"
+	self cumulativePercents 
```

Comment:
Should we keep this method or deprecate it?

Reply:
I do not know. but at least it should be well named.